### ISSUE TITLE
Error with telescope if laravel in subfolder

### ISSUE THREAD

**Issue by vystepanenko** [laravel/telescope#575]
Hello!

I have laravel project that is located in subfolder of main domain
`www.mydomain.com/otherproject/laravelProject`

After installing telescope it opens from this url
`www.mydomain.com/otherproject/laravelProject/public/telescope`

but all links are forward just to  
`www.mydomain.com/telescope/logs`
and ignoring full path to laravel project

I try to play with domain and path in config/telescope.phpp like this 
```
'domain' => 'http://www.mydomain.com',
'path' => '/otherproject/laravelProject/public/telescope',

```
in routes i received this 
```
| mydomain.com | POST     | otherproject/laravelProject/public/telescope/telescope-api/cache                            |           | Laravel\Telescope\Http\Controllers\CacheController@index          | telescope                     |
| mydomain.com | GET|HEAD | otherproject/laravelProject/public/telescope/telescope-api/cache/{telescopeEntryId}         |           | Laravel\Telescope\Http\Controllers\CacheController@show           | telescope                     |
| mydomain.com | POST     | otherproject/laravelProject/public/telescope/telescope-api/commands                         |           | Laravel\Telescope\Http\Controllers\CommandsController@index       | telescope                     |
| mydomain.com | GET|HEAD | otherproject/laravelProject/public/telescope/telescope-api/commands/{telescopeEntryId}      |           | Laravel\Telescope\Http\Controllers\CommandsController@show        | telescope                     |
| mydomain.com | POST     | otherproject/laravelProject/public/telescope/telescope-api/dumps                            |           | Laravel\Telescope\Http\Controllers\DumpController@index           | telescope                     |
| mydomain.com | POST     | otherproject/laravelProject/public/telescope/telescope-api/events                           |           | Laravel\Telescope\Http\Controllers\EventsController@index         | telescope                     |
| mydomain.com | GET|HEAD | otherproject/laravelProject/public/telescope/telescope-api/events/{telescopeEntryId}        |           | Laravel\Telescope\Http\Controllers\EventsController@show          | telescope       
```              
but it dosent helps

**Comment by themsaid** [comment]
Please read all the previous issues on the matter.

TLDR; we don't support running Laravel in a sub folder.
